### PR TITLE
Add missing pt-BR strings

### PR DESCRIPTION
**Pull request by netrik182**
Hi @m2049r. I added the missing strings. Pinging @botoserelepe in case you want them reviewed before merging.

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -366,38 +366,38 @@
...
-    <string name="toast_ledger_start_app">Start Monero App on %1$s</string>
+    <string name="toast_ledger_start_app">Iniciar o App Monero em %1$s</string>
 
-    <string name="menu_rescan">Rescan!</string>
+    <string name="menu_rescan">Reescanear!</string>
 
-    <string name="onboarding_agree">I get it!</string>
-    <string name="onboarding_button_next">Next</string>
-    <string name="onboarding_button_ready">I\'m ready!</string>
+    <string name="onboarding_agree">Eu concordo!</string>
```
Comment (botoserelepe) [inline app/src/main/res/values-pt-rBR/strings.xml]: Maybe "eu entendi" sounds better.

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -366,38 +366,38 @@
...
-    <string name="onboarding_seed_title">Keep your seed safe</string>
-    <string name="onboarding_seed_information">The seed grants full access to whoever has it. If you lose it, we cannot help you recover it and you lose your beloved moneroj.</string>
-    <string name="onboarding_xmrto_title">Send Bitcoin</string>
-    <string name="onboarding_xmrto_information">Monerujo has XMR.to support built-in. Just paste or scan a Bitcoin address and you\'ll be sending BTC by spending XMR.</string>
-    <string name="onboarding_nodes_title">Nodes, your way</string>
-    <string name="onboarding_nodes_information">Nodes connect you to the Monero network. Choose between public nodes or go full cypherpunk using your own.</string>
-    <string name="onboarding_fpsend_title">Send with fingerprint</string>
-    <string name="onboarding_fpsend_information">You\'re now able to send XMR with just your fingerprint if you enabled it. To request the password, just disable fingerprint access.</string>
+    <string name="onboarding_welcome_title">Bem-vindo à Monerujo!</string>
```
Comment (botoserelepe) [inline app/src/main/res/values-pt-rBR/strings.xml]:
```suggestion
    <string name="onboarding_welcome_title">Bem-vindo a Monerujo!</string>
```

**Review by botoserelepe — CHANGES_REQUESTED** [review]
Hey @netrik182 , I just did suggested some small changes ;)

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 2 above)
Comment (netrik182) [reply app/src/main/res/values-pt-rBR/strings.xml]: Good catch, thanks. Added it to the file.

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 1 above)
Comment (netrik182) [reply app/src/main/res/values-pt-rBR/strings.xml]: Yeah, changed.

**Comment by netrik182** [comment]
Let me know if you prefer to have these commits squashed. @m2049r

**Comment by m2049r** [comment]
thank you very much!! yes, it would be better if they're squashed if you have the time @netrik182 :)

**Comment by netrik182** [comment]
Done @m2049r

**Review by m2049r — APPROVED** [review]

**Pull request merged**